app/serializers/localized_ai_persona_serializer.rb

```diff
@@ -3,6 +3,11 @@
 class LocalizedAiPersonaSerializer < ApplicationSerializer
   root "ai_persona"
 
+  def initialize(object, options = {})
+    @features_by_persona_id = options.delete(:features_by_persona_id)
+    super(object, options)
+  end
```

Comment:
yeah, this is a bit of a weird pattern I can refine it today so we don't need this override.